src/MeshKernelApi/MeshKernel.cpp

```diff
@@ -123,8 +124,7 @@ namespace meshkernelapi
 
         for (const auto& index : indices)
         {
-            const auto size = index[1] - index[0] + 1;
-            if (size > 0)
+            if (const auto size = index[1] - index[0] + 1; size > 0)
```

Comment:
I still don't like this syntax...

Reply:
I see your point if I can't inspect the value of size when debugging. if this is the case we can revert this